Makefile

```diff
@@ -214,21 +214,12 @@ jsonnet: ## Download jsonnet locally if necessary.
 
 GINKGO = $(shell pwd)/bin/ginkgo
 ginkgo: ## Download ginkgo and gomega locally if necessary.
-	$(call go-get-tool,$(GINKGO),github.com/onsi/ginkgo/v2/ginkgo)
-	$(call go-get-tool,$(GOMEGA),github.com/onsi/gomega/...)
+	$(call go-get-tool,$(GINKGO),github.com/onsi/ginkgo/v2/ginkgo@v2)
```

Comment:
Should this use the latest version available? Currently it is https://github.com/onsi/ginkgo/releases/tag/v2.1.6